lib/srv/reexec.go

```diff
@@ -117,7 +123,7 @@ type ExecCommand struct {
 	// UaccMetadata contains metadata needed for user accounting.
 	UaccMetadata UaccMetadata `json:"uacc_meta"`
 
-	// X11Config contains an xauth entry to be added to the command user's xauthority.
+	// X11Config contains a xauth entry to be added to the command user's xauthority.
```

Comment:
grammar nit: "an xauth" is actually correct here, since the "x" is pronounced like a vowel "ex".